### ISSUE TITLE
Track Change enabled when document is shared

### ISSUE THREAD

**Issue by p4pr8n** [ONLYOFFICE/onlyoffice-nextcloud#2]
Hello,
I don't know if it is a real issue but as found it in ONLYOFFICE doc.
When a document is shared the track change option should be automatically set.
Thanks,
p4pr8n

**Comment by mr-bolle** [comment]
i thing they currently migrate onlyoffice to nextcloud
here you can find the documentation. https://api.onlyoffice.com/editors/owncloud

**Comment by p4pr8n** [comment]
@mr-bolle I am using onlyoffice from Nextcloud (last version for both). I don't understand your comment.

**Comment by HelenBelova17** [comment]
Hello, **@p4pr8n**, please send us a screenshot with the track change option for better understanding what you mean.

**Comment by p4pr8n** [comment]
Hello, @HelenBelova17, as found in this link (see note),

https://helpcenter.onlyoffice.com/ONLYOFFICE-Editors/ONLYOFFICE-Document-Editor/HelpfulHints/Review.aspx

_Enable the Track Changes feature
To see changes suggested by a reviewer, enable the Track Changes option in one of the following ways:
click the Track changes button button in the right lower corner at the status bar, or
switch to the Collaboration tab at the top toolbar and press the Track Changes button Track Changes button.
_Note: it is not necessary for the reviewer to enable the Track Changes option. It is enabled by default and cannot be disabled when the document is shared with review only access rights.__

In Nextcloud, when I share a document, the Track Changes option is not enabled by default.
This option must be enabled manually by each participant.

I can of course send a screenshot if still needed.

**Comment by alexanderonlyoffice** [comment]
Hello @p4pr8n, unfortunately, there is no possibility in Nextcloud to share files with only review permissions. We cannot change Nextcloud share feature, that's why every participant should enable it manually in editor. It means that full editor is being opened by default, where one can enable Review mode or disable it.

**Comment by alexanderonlyoffice** [comment]
@p4pr8n, yes, Nextcloud share feature does not allow to do it with ONLY review permissions.

**Comment by p4pr8n** [comment]
@alexanderonlyoffice Could it be possible (in future) to implement a work around in onlyoffice-nexcloud to force the use "ONLY review" when a document is shared (option in onlyoffice-nextcloud settings) ? Thanks

**Comment by alexanderonlyoffice** [comment]
@p4pr8n, we will take it into consideration.

**Comment by p4pr8n** [comment]
Thanks a lot.

**Comment by p4pr8n** [comment]
More information. When a document was created within MsWord with the equivalent of "track change" enabled, this document will open with track change enabled in Netcloud-ONLYOFFICE. I deduce that this state can maybe be saved in the edited file ?

**Comment by tiagoefreitas** [comment]
So there should be way to enable track change on the document level in the UI, not just by converting a docx file

**Comment by LinneyS** [comment]
Hi @p4pr8n 
We are discussing with Nextcloud the possibility of expanding the list of rights.
For issue `the equivalent of "track change" enabled` look at the [DocumentServer](https://github.com/ONLYOFFICE/DocumentServer) repository

**Comment by el-calavera** [comment]
It does not have to be that complicated, no new features are required.
Onlyoffice only needs to heed the "track changes" feature, which is part of the DOCX format specification anyway.
That way, collaborators will automatically start with "track changes" enabled if it's enabled in the file, and you get a behaviour comparable to other collaborative editing software like Etherpad or Google Docs.

**Comment by modaresimr** [comment]
no any progress on that?
